config_utils.py

```diff
@@ -3,7 +3,7 @@
 
 
 def load_config():
-    config_file = r'data_location.config'
+    config_file = r'data_location.template.config'
```

Comment:
Please revert to `data_location.config`.
The workflow if the following:
- `data_location.template.config` is in git. It gives you an exemple of the paths you must write in your config, but it is **not** the config itself.
- in your local repo, copy `data_location.template.config` and rename it `data_location.config`. Change the path depending on where you stored the data.
- `data_location.config` is in `.gitignore` so nobody commits its local config.

This is required because Stepan is working on Windows and which doesn't have the same file system than us.

```suggestion
    config_file = r'data_location.config'
```

Reply:
Ok I didn’t understand that correctly, this could be fixed by using only pathlib.Path objects.

Reply:
Yes, but let's do it in another PR :)
I am committing my suggestion to make sure you doesn't forget it!